scripts/superdesk-search/search.js

```diff
@@ -2248,9 +2248,18 @@
          * Multiple item spike
          */
         this.spikeItems = function() {
-            spike.spikeMultiple(multi.getItems());
-            $rootScope.$broadcast('item:spike');
-            multi.reset();
+            if ($location.path() === '/workspace/personal') {
+                modal.confirm(gettext('Do you want to delete the item permanently? '))
```

Comment:
For personal workspace there isn't unspike, so action is permanent. 

For other comments, I will fix it as you suggested ;)